src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -1,88 +1,69 @@
+using MarkdownLog;
+using Microsoft.Xunit.Performance.Api.Table;
 using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.IO;
 using System.Linq;
-using System.Reflection;
-using Microsoft.Xunit.Performance;
-using Microsoft.Xunit.Performance.Api.Table;
+using System.Xml.Linq;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
     public sealed class XunitPerformanceHarness : IDisposable
     {
-        private string[] _args;
-
         public XunitPerformanceHarness(string[] args)
         {
             _args = args;
+            _disposed = false;
 
             // Set the run id.
-            Configuration.RunId = DateTimeOffset.UtcNow.ToString("yyyy-MM-dd-HH-mm-ss");
+            Configuration.RunId = $"{DateTimeOffset.UtcNow.ToString("yyyyMMddHHmmss")}";
 
             // Set the file log path.
             // TODO: Conditionally set this based on whether we want a csv file written.
             Configuration.FileLogPath = Configuration.RunId + ".csv";
```

Comment:
I think this should go away eventually.

---
In reply to: [100348024](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100348024) [](ancestors = 100348024)